DataConnectors/Forcepoint CASB.json

```diff
@@ -0,0 +1,108 @@
+            "description": "Install and configure the Linux agent to collect your Common Event Format (CEF) Syslog messages and forward them to Azure Sentinel.\n\n> Notice that the data from all regions will be stored in the selected workspace. ",
+            "innerSteps": [
+                {
+                    "title": "1.1 Select or create a Linux machine",
+                    "description": "Select or create a Linux machine that Azure Sentinel will use as the proxy between your security solution and Azure Sentinel this machine can be on your on-prem environment, Azure or other clouds."
+                },
+                {
+                    "title": "1.2 Install the  Forcepoint CASB CEF collector on the Linux machine",
+                    "description": "To Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector collects CEF messages on port 514 TCP.\n\n>Steps to install and configure Forcepoint CASB connector are provided in the Forcepoint CASB integration installation guide which can be found in this link:\nhttps://support.forcepoint.com/knowledgebase#sort=relevancy\n\n",
+                    "instructions": [
+                        {
+                            "parameters": {
+                                "fillWith": [
+                                    "WorkspaceId",
+                                    "PrimaryKey"
+                                ],
+                                "label": "Copy the following command and insert it into the settings.yml file as explained in the above link",
+                                "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+                            },
+                            "type": "CopyableLabel"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "title": "2. Forward the steps in Forcepoint CASB integration installation guide to complete the installation",
+            "description": "Once the installation is finished you can do the next step to validate the connection between Forcepoint CASB  and your Azure Sentinel workspace "
+        },
+        {
+            "title": "3. Validate connection",
+            "description": "Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python –version\n\n>2. You must have elevated permissions (sudo) on your machine",
```

Comment:
I see formatting issues when I load this json file using the 'import' test functionality specifically after 'python -version\n\n/...' Fix this by deleting the -version\n and just type out '-version \n...' by adding a space between version and \n. Check using the import tool to validate.